### PR TITLE
Renamed Vset.modules to Vset.vfuncs and Vset.out to Vset.fitted_vfuncs

### PR DESCRIPTION
**Pull request by ssaxena00**
- Still have to change docs (how to do this?)
- Can still change:
   - [ ] ‘apply_modules’ -> ‘apply_vfuncs’ (in utils.py), 
   - [ ] ‘modules’ -> ‘funcs’ (in vset.py)
   - [ ] 'module_keys' -> 'func_keys' (in vset.py)

### CODE REVIEW

File: vflow/vset.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def __init__(self, name: str, modules, module_keys: list = None,
         if not self._async and np.any([isinstance(mod, AsyncModule) for mod in modules]):
             self._async = True
         if isinstance(modules, dict):
-            self.modules = modules
+            self.vfuncs = modules
```
Comment (jpdunc23) [inline vflow/vset.py:74]: Let's also change the init argument names from `modules` and `module_keys` to `funcs` and `func_keys`.

**Review by jpdunc23 — CHANGES_REQUESTED** [review]

**Comment by ssaxena00** [comment]
Discovered errors in p_check.ipynb:
- removed `compute_Interval` ?
- in executing `filtered_rf_set.fit(X_train_clean, y_train)`:
  - got `TypeError: float() argument must be a string or a number, not 'dict'`

**Comment by ssaxena00** [comment]
Discovered errors in `build_Vset.ipynb`:
- call to `build_vset` is not resulting in correct `.vfuncs` attribute
   - length is 1 and vfunc looks like:
   `RandomForestRegressor(criterion='absolute_error',
                      n_estimators={'max_features': ['sqrt', 'log2'],
                                    'min_samples_split': [2, 10],
                                    'n_estimators': [100, 200, 300]})`

**Comment by ssaxena00** [comment]
Discovered errors in `03_computer_vision_dnn`:
- while executing `modeling_set.fit(train_data)`:
   - got `TypeError: train() got multiple values for argument 'loss_fn'`

**Comment by ssaxena00** [comment]
Causing error in `04_feat_importance_stability.ipynb` because `build_vset` not setting `.vfuncs` attribute correctly

**Comment by ssaxena00** [comment]
Discovered error in `mlflow_logging_hardcoded`: 
- while executing `train(C, min_samples_leaf`:
   - got `KeyError: ('X_test', (X_train, X_train, LR, 'X_test'), 'Acc')`

File: vflow/vset.py (hunk 1 above)
Comment (ssaxena00) [reply vflow/vset.py:74]: Should I also change  `apply_modules` -> `apply_vfuncs` (in utils.py)?

Comment (jpdunc23) [reply vflow/vset.py:74]: Yes, please. Let's replace all usage of `module` with `vfunc`.

**Comment by jpdunc23** [comment]
> Discovered error in `mlflow_logging_hardcoded`:
> 
> * while executing `train(C, min_samples_leaf`:
>   
>   * got `KeyError: ('X_test', (X_train, X_train, LR, 'X_test'), 'Acc')`

@ssaxena00  We can delete that notebook. Also feel free to replace `compute_interval` with `perturbation_stats` and let me know if usage of the latter isn't clear.

**Pull request closed without merging**